docker-compose.yml

```diff
@@ -132,7 +144,7 @@ services:
   statsd:
     image: hopsoft/graphite-statsd
     ports:
-      - "8080:80"
+      - "8081:80"
```

Comment:
I would change this container since `oidcprovider` would be taking port `8080`

Reply:
I see, thanks for the explanation. I feel like this might be simpler so I am opting to leaving it at is. I am up for changing if it becomes a problem though.